tasks/prime_workflow.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+import logging
+import random
+
+from locust import tag, task
+
+from .prime import PrimeTasks, SupportTasks
+from utils.constants import (
+    MOVE_TASK_ORDER,
+    MTO_SHIPMENT,
+    MTO_SERVICE_ITEM,
+    PAYMENT_REQUEST,
+)
+from utils.base import ImplementationError
+
+logger = logging.getLogger(__name__)
+
+
+@tag("workflow")
+class WorkflowTasks(PrimeTasks, SupportTasks):
+    """Workflow Task Set
+    Each task is a workflow from start to finish. Each workflow acts on only one MTO from start to finish.
+    There are multiple workflows for different scenarios such as a single shipment HHG move, or an HHG move with a SIT.
+    Multiple instances of the tasks will run during load testing and will interleave their hits to the server.
+    However they will run in sequence and should all complete without any errors.
+    """
+
+    # current_move is the move the whole task will be acting on
+    # all updates to the move and nested objects will be stored to this mto
+    current_move = None
+    # workflow_descriptor
+    workflow_descriptor = None
```

Comment:
Is this ever used?

Reply:
oops, it was supposed to be workflow_title.